image/seafile_8.0/Dockerfile

```diff
@@ -0,0 +1,62 @@
+# See https://hub.docker.com/r/phusion/baseimage/tags/
+FROM phusion/baseimage:0.11
+ENV SEAFILE_SERVER=seafile-server SEAFILE_VERSION=
+
+RUN apt-get update --fix-missing
+
+# Utility tools
+RUN apt-get install -y vim htop net-tools psmisc wget curl git
+
+# For suport set local time zone.
+RUN export DEBIAN_FRONTEND=noninteractive && apt-get install tzdata -y
+
+# Nginx
+RUN apt-get install -y nginx
+
+#
+RUN apt-get install -y libmysqlclient-dev
+
+# Python3
+RUN apt-get install -y python3 python3-pip python3-setuptools
+RUN python3.6 -m pip install --upgrade pip && rm -r /root/.cache/pip
+
+RUN pip3 install --timeout=3600 click termcolor colorlog pymysql \
+    django==2.2.* && rm -r /root/.cache/pip
+
+RUN pip3 install --timeout=3600 future mysqlclient Pillow pylibmc captcha jinja2 \
+    sqlalchemy django-pylibmc django-simple-captcha && \
```

Comment:
can you please consider adding pyjwt here since that would allow closing #231 ? thx

Reply:
Thanks for helping us find the problem.